spec/type_aliases/filesource_spec.rb

```diff
@@ -25,8 +46,6 @@
           '\\Users/hc/wksp/stdlib',
           'C:noslashes',
           '\\var\\tmp',
-          'puppet:///foo/bar.log',
-          'puppet:///pm.example.com/modules/foo/bar.log',
```

Comment:
At first glance this might not look valid, but I can't see anything that says `pm.example.com` couldn't be a valid mount point.  Unlikely and unusual, yes, but not technically invalid.